backend/flyway/build_flyway_image.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
-mkdir -p migration
-cp ../src/main/resources/db/migration/*.sql migration/
+mkdir -p migration_tmp
+cp ../src/main/resources/db/migration/*.sql migration_tmp/
 docker build . -t trackyourbed-flyway
```

Comment:
Can we actually make the tag an argument.
If you don't wanna do that, I can take care of this in a separate MR 😉

Reply:
That's a good suggestion, then the parameters the local setup relies on to be the same are closer to each other, I will add it here

Reply:
pushed

Reply:
Awesome 👍